### ISSUE TITLE
[Stats Perform] Deduce receiving player information for Pass event 

### ISSUE THREAD

**Issue by DriesDeprest** [PySport/kloppy#249]
Unlike other data providers, Stats Perform does not give information about the pass receiver in the raw data of their pass events.

We could, however, deduce the receiving player ID from the next event. If we consider the next event after a successful pass and use the ID of the player executing that event, if the next event is a ball owning event.

@koenvo do you think this is a good idea?

**Comment by probberechts** [comment]
Yes, this is a good idea. If you can *reliably infer* who is the intendend recipient of a pass, you can impute the `receiver_player`. It probably requires a few more checks though (e.g., filter out deflected passes, passes interrupted by a foul, etc.).
